/*
    let & const II

*/

try {
    if (true) {
        const NUM = 10;

        console.log(NUM);
    }
} catch (e) {
    console.error("error: " + e);
}

